### PR TITLE
*: rename NextChunk to Next

### PR DESCRIPTION
**Pull request by XuHuaiyu**
rename NextChunk to Next
PTAL @zz-jason @winoros

### CODE REVIEW

File: executor/aggregate.go [hunk 1]
```diff
@@ -110,11 +110,11 @@ func (e *HashAggExec) NextChunk(ctx context.Context, chk *chunk.Chunk) error {
 	}
 }
 
-// innerNextChunk fetches Chunks from src and update each aggregate function for each row in Chunk.
+// innerNext fetches Chunks from src and update each aggregate function for each row in Chunk.
```
Comment (zz-jason) [inline executor/aggregate.go]: s/innerNext/execute/

**Comment by XuHuaiyu** [comment]
PTAL @zz-jason

**Pull request merged**